tests/web/test_assets.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+ASSETS_ENDPOINT = '/api/v1/assets/'
+
+
+def test_get_assets_with_empty_text_search(client):
+    res = client.get(ASSETS_ENDPOINT + '?search=')
+    assert res.json == {'status': 400,
+                        'message': 'text_search cannot be empty'}
+    assert res.status_code == 400
+
+
+def test_get_assets_with_missing_text_search(client):
+    res = client.get(ASSETS_ENDPOINT)
+    assert res.status_code == 400
+
+
+@pytest.mark.bdb
+@pytest.mark.genesis
```

Comment:
When using the `genesis` marker the `bdb` marker is internally invoked, so it is not explicitly needed. See https://github.com/bigchaindb/bigchaindb/blob/master/tests/conftest.py#L186 -- the `_bdb` fixture is used by the `bdb` marker (https://github.com/bigchaindb/bigchaindb/blob/master/tests/conftest.py#L71-L74)